src/static/css/index.css

```diff
@@ -1,8 +1,14 @@
 body {
-  background-image: url(/static/images/intro-background-fit.png);
   background-repeat: no-repeat;
   background-size: 100% 910px;
 }
+
+body.lazy {
+  /* This is a 1px solid #677486 image. */
+  background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNML2n7DwAFDQJi8nQACgAAAABJRU5ErkJggg==);
```

Comment:
This is not `background-color` because then it ignores the `background-size` attribute, and stretches the full length of the page and looks bad.